content/docs/user-guide/glossary/workspace.md

```diff
@@ -6,3 +6,6 @@ match: [workspace]
 Directory containing all your project files e.g. raw datasets, source code, ML
 models, etc. Typically, it's also a Git repository. It will contain your DVC
 project.
+
+Learn more about the [workspace concept](/doc/user-guide/concepts/workspace) in
+DVC.
```

Comment:
Try to stick to a single paragraph for tooltips, check out how they look in the review app. That last sentence for the link to basic concepts can probably be shorter too.

Reply:
And I guess we're adding similar links to all tooltip texts?

Reply:
But wait... I'm confused. I thought we were keeping a single file per concept with both the `glossary` text in Frontmatter, and the md content for the actual docs?

Reply:
Agreed on single paragraph. For the links I think it will come down to which items have their own page (basic concepts) and which ones remain as glossary items (same as previous behaviour). Yes, a single file per concept.